.circleci/config.yml

```diff
@@ -23,7 +23,18 @@ jobs:
             # Avoid an error `linux mounts: path /etc/kubernetes is mounted on / but it is not a shared or slave mount`
             sudo mount --make-shared /
             # TODO: remove the --bootstrapper flag once this issue is solved: https://github.com/kubernetes/minikube/issues/2704
-            sudo -E minikube start --vm-driver=none --bootstrapper=localkube --kubernetes-version=${K8S_VERSION} --extra-config=apiserver.Authorization.Mode=RBAC
+            sudo -E minikube start --vm-driver=none --bootstrapper=localkube --memory=3072 --kubernetes-version=${K8S_VERSION} --extra-config=apiserver.Authorization.Mode=RBAC --extra-config=kubelet.PodCIDR=192.168.0.0/16 --extra-config=controller-manager.cluster-cidr=192.168.0.0/16 --extra-config=controller-manager.allocate-node-cidrs=true --network-plugin=cni --extra-config=kubelet.network-plugin=cni
+      - run:
+          name: Make /run shared mount
+          command: |
+            # Otherwise calico-node will fail to start with:
+            # Error: failed to start container "calico-node": ... path /var/run/calico is mounted on /run but it is not a shared or slave mount
+            sudo mount --make-shared /run
+      - run:
+          name: Install calico plugin
+          command: |
+            kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/rbac.yaml
```

Comment:
Can you try with the latest version, 3.2? Also, can you add a comment with a link to the documentation for future readers?